packages/node-resolve/src/index.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable no-param-reassign, no-shadow, no-undefined */
 import { dirname, normalize, resolve, sep } from 'path';
 
-import isBuiltinModule from 'is-builtin-module';
+import { builtinModules } from 'module';
```

Comment:
Won't this cause a behavior change if you're bundling on one version of node, but a newer node has that module available? Or does it not matter because node will never add any new module without the `node:` prefix?

Reply:
It's unlikely they will add any new modules without the prefix as far as I understand

Also the third party package also has a fixed list which won't always be accurate to the runtime node